metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
```

Comment:
Yeah, I thought about that too but forgot about it and used "Metric". I'll change it